Remove text specifying that app service plan can only be shared by ap…

andynava opened the pull request:
…ps w/in same resource group

It is possible to have apps share an App Service Plan that is in a different resource group.

Scenario:
Resource Group: resourcegroup-01
App Service Plan: plan-in-resourcegroup-01
Web App: webapp-in-resourcegroup-01

New API App...
Resource Group: resourcegroup-02
API App: apiapp-in-resourcegroup-02
App Service Plan: plan-in-resourcegroup-01

azurecla commented:
Hi __@fashah__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (fashah). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

andynava committed:
Remove text specifying that app service plan can only be shared by apps w/in same resource group

It is possible to have apps share an App Service Plan that is in a different resource group.

Scenario:
Resource Group: resourcegroup-01
App Service Plan: plan-in-resourcegroup-01
Web App: webapp-in-resourcegroup-01

New API App...
Resource Group: resourcegroup-02
API App: apiapp-in-resourcegroup-02
App Service Plan: plan-in-resourcegroup-01
